src/include/rbd/librbd.h

```diff
@@ -641,6 +641,11 @@ CEPH_RBD_API int rbd_mirror_image_get_status(rbd_image_t image,
                                              rbd_mirror_image_status_t *mirror_image_status,
                                              size_t status_size);
 
+// RBD consistency groups support functions
+CEPH_RBD_API int rbd_create_cg(rados_ioctx_t p, const char *name);
```

Comment:
Mixed feelings about "cg" naming style in the API since it really isn't self-describing.  Probably can't justify a name as long as "rbd_consistency_group_snap_remove" --- but maybe "rbd_group_XYZ"?